docs/guides/deploy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Deploying your first project with Nitric Deploy
+description: Nitric Deploy builds and deploys the right infrastructure from your code in your chosen cloud.
+---
+
+## What we'll be doing
+
+1. Sign to Nitric Deploy
+2. Create an organization
+3. Set up AWS credentials
+4. Create or import a project
+5. Create an environment
+6. Configure environment variables
+7. Deploy your project
+8. Check out the logs
+
+<br/>
+
+## Sign to Nitric Deploy
+
+> Nitric Deploy is currently in Alpha, you'll need to [request access](https://deploy.nitric.io/signup).
```

Comment:
change "Alpha" to "preview" ?

Reply:
The comma in the middle of this should be a period. 

Let's also make the second sentence, "[Request access](https://deploy.nitric.io/signup) using your Github account email address."